openquake/calculators/disaggregation.py

```diff
@@ -123,9 +123,10 @@ def compute_disagg(dstore, slc, cmaker, iml2s, trti, bin_edges, monitor):
         oq.investigation_time)
     pne_mon = monitor('disaggregate_pne', measuremem=False)
     mat_mon = monitor('build_disagg_matrix', measuremem=False)
+    gmf_mon = monitor('computing mean_std', measuremem=False)
```

Comment:
`gmf` is a bit misleading. What about gms?

Reply:
I am not sure what to use here. I will think of something.